NitroxClient/GameLogic/Spawning/DefaultEntitySpawner.cs

```diff
@@ -25,20 +25,24 @@ public Optional<GameObject> Spawn(Entity entity, Optional<GameObject> parent)
 
             GameObject gameObject = Utils.SpawnFromPrefab(prefab, null);
             gameObject.transform.position = entity.Position;
+            gameObject.transform.rotation = entity.Rotation;
             gameObject.transform.localScale = entity.Scale;
 
             if (parent.IsPresent())
             {
-                gameObject.transform.SetParent(parent.Get().transform, true);
+                gameObject.transform.SetParent(parent.Get().transform, false);
             }
 
-            gameObject.transform.localRotation = entity.Rotation;
             gameObject.SetActive(true);
 
-            NitroxIdentifier.SetNewId(gameObject, entity.Id);
-            LargeWorldEntity.Register(gameObject);
+            NitroxEntity.SetNewId(gameObject, entity.Id);
             CrafterLogic.NotifyCraftEnd(gameObject, techType);
 
+            if (parent.IsPresent() && parent.Get().GetComponent<LargeWorldEntityCell>())
```

Comment:
Are there any other types of entities that need to be registered?  Wasn't this previously causing orphaned game objects that need to be cleaned up?

Reply:
Previously it was but no more

Reply:
This is just registering everything thats a direct child of the EntityCell no reason to register the children of the child as well